sdk/notificationhubs/notification-hubs/review/notification-hubs.api.md

```diff
@@ -284,7 +308,7 @@ export function createAdmTemplateRegistrationDescription(description: AdmTemplat
 export function createAppleInstallation(installation: DeviceTokenInstallation): AppleInstallation;
 
 // @public
-export function createAppleNotification(notification: NotificationCommon): AppleNotification;
+export function createAppleNotification(notification: AppleNotificationCommon): AppleNotification;
```

Comment:
Feels a little weird that it's called AppleNotification**Common** still even though it's no longer common, it's specific to Apple.

Reply:
Renamed to params